### DISCUSSION TITLE
Puddleglum

### DISCUSSION THREAD

**Discussion by jonathanminson** [HassanZahirnia/laravel-package-ocean#41] [Package Suggestions]
https://github.com/calvient/puddleglum

Puddleglum allows you to automatically generate TypeScript interfaces from your Laravel models and requests as well as a typed Axios implementation for your API routes.

**Comment by HassanZahirnia** [comment]
Hey @jonathanminson 👋

Thanks for your suggestion! I don't mind adding your package since I personally am a Typescript fan 💙, but it doesn't seem to be compatible with Laravel 10, If you do something about that I can add your package to the database.
